vita3k/gui/include/gui/functions.h

```diff
@@ -36,14 +36,15 @@ enum GenericDialogState {
     CANCEL_STATE
 };
 
-void delete_game(GuiState &gui, HostState &host, const std::string &title_id);
-void game_context_menu(GuiState &gui, HostState &host, bool *selected, const std::string &title_id);
+void delete_game(GuiState &gui, HostState &host);
+void game_context_menu(GuiState &gui, HostState &host, bool *selected);
 void get_game_titles(GuiState &gui, HostState &host);
 void get_modules_list(GuiState &gui, HostState &host);
 void init(GuiState &gui, HostState &host);
 void init_background(GuiState &gui, const std::string &image_path);
 void init_icons(GuiState &gui, HostState &host);
-void load_game_background(GuiState &gui, HostState &host, const std::string &title_id);
+void load_game_background(GuiState &gui, HostState &host);
+void load_manuals(GuiState &gui, HostState &host, const std::string &image_name);
```

Comment:
Keep the string parameter but just pass `host.io.title_id`.

Reply:
don't needed, is good like that

Reply:
What if someone needs to load an image of the next game? Or a newly added game? They would  have to store `host.io.title_id` in some temp variable, set it again to their preferred game then call the function and restore it from the temp variable. Just pass `host.io.title_id` instead of hard wiring every function to use it. It leaves options open.

Reply:
like i said, don't have any problem, value is updated and perfecly works for all

Reply:
Maybe for now but someone in the future might have to revert this- I really don't see why you can't just pass it.

Reply:
because is not needed, string title id is used only if you don't have value outside, but here is already exist on io.title id, so not needed